userguide/tn_initial.rst

```diff
@@ -118,13 +118,21 @@ before the license key for the HA %brand% system can be entered. The key
 is entered on the active |ctrlr-term|. Click :guilabel:`OK` to save the
 license key and access the |web-ui|.
 
-.. _tn_initial:
+.. _tn-dashboard:
 
-.. figure:: images/truenas/initial1c.png
+.. figure:: images/truenas/dashboard.png
 
-   %brand% Graphical Configuration Menu
+   %brand% Dashboard
 
 
+The %brand%
+:menuselection:`Dashboard`
+shows details about the system. These details are grouped into
+sections about the |ctrlrs-term|, hardware components, networking,
+storage, and other categories. The |ctrlr-term| sections include
+buttons to :guilabel:`CHECK FOR UPDATES` or
+:guilabel:`INITIATE FAILOVER` in :ref:`HA <Failover>` systems.
+
 If the storage devices are encrypted, a prompt requests the encryption
```

Comment:
Need some kind of header to make it clear that this next paragraph is a new subject (disk encryption). It should also mention when the prompt for the passphrase appears.  On first connect?  On attach?